src/config.h.in

```diff
@@ -51,7 +51,7 @@
 #define GMT_BINARY_DIR_SRC_DEBUG "@GMT_BINARY_DIR@/src"
 
 /* path to documentation */
-#define GMT_DOC_DIR "@CMAKE_INSTALL_PREFIX@/@GMT_DOCDIR@"
+#define GMT_DOC_DIR "@GMT_DOCDIR@"
```

Comment:
I'm a little confused here. Does it mean `@GMT_DOCDIR@` must be an absolute path?

Reply:
Well, the way it was (line two above) you got two hard paths added together.

Reply:
Yes, but if someone gives GMT_DOCDIR a relative path, then `GMT_DOC_DIR` is also a relative path.

Reply:
Well, I can put it back to what it was but then we get line 2 for brew.
We could add a GMT_INTALLDIR="@CMAKE_INSTALL_PREFIX@" to config.h to have a separate parameter for that, but I guess I don't understand exactly how the package managers are specifying things like doc dir, i.e., relative (to what?) or absolute.

Reply:
Homebrew uses `-DGMT_DOCDIR=#{share}/doc/gmt`. I don't know exactly what `#{share}` means in homebrew, but probably homebrew should use `-DGMT_DOCDIR=share/doc/gmt`, which is relative to `${CMAKE_INSTALL_PREFIX}`.